improver_tests/nbhood/circular_kernel/test_GeneratePercentilesFromACircularNeighbourhood.py

```diff
@@ -193,10 +207,14 @@ def test_irregular_kernel(self):
 
     def test_single_point_almost_edge(self):
         """Test behaviour for a non-zero grid cell quite near the edge."""
-        cube = set_up_cube(
-            zero_point_indices=[(0, 0, 1, 1)], num_grid_points=3
-        )  # Just within range of the edge.
-        slice_2d = cube[0, 0, :, :]
+
+        data = np.ones((3, 3), dtype=np.float32)
+        data[1, 1] = 0
+        cube = set_up_variable_cube(data, spatial_grid="equalarea",)
+
+        # Just within range of the edge.
+
+        slice_2d = cube[:, :]
```

Comment:
The cube is already 2D.  What effect does this statement have?

Reply:
No effect now, apologies for the oversight, it used to be a 3D cube with a realization of 1 which was useless so I removed it.